core/src/main/scala/org/broadinstitute/dsde/rawls/model/SamModel.scala

```diff
@@ -88,6 +88,7 @@ object SamBillingProjectActions {
   val own = SamResourceAction("own")
   def sharePolicy(policy: String) = SamResourceAction(s"share_policy::$policy")
   def readPolicy(policy: SamResourcePolicyName) = SamResourceAction(s"read_policy::${policy.value}")
+  def link = SamResourceAction("link")
```

Comment:
nit: `val`, not `def` (the compiler probably fixes this for you, but I'm not cure)